index.bs

```diff
@@ -6714,6 +6730,52 @@ During a transition from the FIDO U2F JavaScript API, a [=[RP]=] may have a popu
 : Authenticator extension output
 :: None.
 
+### Conditional Create Extension (<dfn>conditionalCreate</dfn>) ### {#sctn-authenticator-conditional-create-extension}
+
+This [=client extension|client=] [=authentication extension=] indicates that the RP would like to create a credential after an authenticaton ceremony is successfully mediated by the user agent.
+
+
+: Extension identifier
+:: `conditionalCreate`
+
+: Operation applicability
+:: [=authentication extension|Authentication=]
+
+: Client extension input
+:: The Boolean value [TRUE] to indicate that this extension is requested by the [=[RP]=].
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+        boolean conditionalCreate;
+    };
+    </xmp>
+
+: Client extension processing
+
+:: When [[[#sctn-getAssertion|assertion]]:
+        1. Just after [establishing the RP ID](#GetAssertion-DetermineConditional) perform these steps:
+
+            1. If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is not present or does not have the value
+     {{CredentialMediationRequirement/conditional}}:
+
+                1. Throw a "{{NotAllowedError}}" {{DOMException}}
+
+        1. When [showing conditional assertion form control](GetAssn-ConditionalMediation-Interact-FormControl):
+
+            1. Inform the user that the RP will create a credential if the user agent mediates the authentication ceremony.
+
+            1. Set conditionalCreateLifetimeTimer to a client-specific default.
```

Comment:
```suggestion
            1. Set |conditionalCreateLifetimeTimer| to a client-specific default.
```

(Your call. I think we've put bars around variable names elsewhere in the spec but I'm not sure that it's done perfectly consistently.)

Reply:
For the record, this has some benefits beyond visual style: this [annotates it semantically as a variable](https://speced.github.io/bikeshed/#var-and-algorithms). Bikeshed does some minimal validation, like warning if a variable is defined but never referenced.

Reply:
Right, and it does :slightly_smiling_face:

```
LINE ~1920: The var 'conditionalCreateLifetimeTimer' (in global scope) is only used once.
If this is not a typo, please add an ignore='' attribute to the <var>.
LINE ~1920: The var 'conditionalCreateOrigin' (in global scope) is only used once.
If this is not a typo, please add an ignore='' attribute to the <var>.
```